src/v2/spec.yaml

```diff
@@ -17,6 +17,16 @@ components:
       description: "API key to authorize requests"
       x-default: "NEYNAR_API_DOCS"
 
+  parameters:
+    NeynarExperimentalHeader:
+      name: x-neynar-experimental
+      in: header
+      required: false
+      schema:
+        type: boolean
+      x-default: false
```

Comment:
@valerierose 
Wanna understand why a `x-default` is used here.
```
schema:
        type: boolean
        default: false
```
If we do this it'll work, I just tested it